CDCgov/data-exchange-fhir
# Lambda function for saving logs to S3 bucket

**Additions**:
Created a Lambda function that exports Logs to a S3 Bucket.

**Removals**
None

**Updates**
New Lambda Function

**Testing Steps**

- Run the application locally
- Run the Lambda function in Digital HIE AWS account. 
- Make sure that we getting a good response back as displayed below.
- Check that logs are exported into the S3 bucket indicated.

**Notes**
- Permission and other information provided in ReadMe
